pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -3211,6 +3230,59 @@ func TestListPlugins(t *testing.T) {
 	}
 }
 
+func TestClose(t *testing.T) {
+	tests := []struct {
+		name    string
+		plugins *config.Plugins
+		wantErr string
+	}{
+		{
+			name: "close doesn't return error",
+			plugins: &config.Plugins{
+				MultiPoint: config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: testPlugin, Weight: 5},
+					},
+				},
+			},
+		},
+		{
+			name: "close returns error",
+			plugins: &config.Plugins{
+				MultiPoint: config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: testPlugin, Weight: 5},
+						{Name: testCloseErrorPlugin},
+					},
+				},
+			},
+			wantErr: "test-close-error-plugin failed to close: close err",
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			_, ctx := ktesting.NewTestContext(t)
+			ctx, cancel := context.WithCancel(ctx)
+			defer cancel()
+			fw, err := NewFramework(ctx, registry, &config.KubeSchedulerProfile{Plugins: tc.plugins})
+			if err != nil {
+				t.Fatalf("Unexpected error during calling NewFramework, got %v", err)
+			}
+			closeErr := fw.Close()
+			if closeErr != nil {
+				if diff := cmp.Diff(closeErr.Error(), tc.wantErr); diff != "" {
```

Comment:
this is still comparing strings... do not compare strings, compare errors.

Reply:
/hold

Reply:
It should look something like this `cmp.Diff(closeErr, tc.wantErr, cmpopts.EquateErrors())`. Then there is no need to check `closeErr != nil`. That will implicitly be checked by diff.

Reply:
@alculquicondor,
Thanks for your comment!
I tried implementing the following code, but it didn't work because the pointers within errors.Join() are different:

```go
wantErr: errors.Join(fmt.Errorf("%s failed to close: %w", "test-close-error-plugin", errors.New("close err"))),
...
if diff := cmp.Diff(closeErr, tc.wantErr, cmpopts.EquateErrors()); diff != "" {
    t.Fatalf("Unexpected error during calling Close(). (-want,+got):%s", diff)
}
```

Reply:
@alculquicondor,
As a result, I implemented using the check `closeErr != nil`.
If you have any other good ideas, could you let me know?
Thanks,

Reply:
That's easily solvable by using the same pointer :)